src/app/containers/StoryPromo/LinkContents.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react';
+import moment from 'moment-timezone';
+import { shape } from 'prop-types';
+
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+
+import { ServiceContext } from '../../contexts/ServiceContext';
+import deepGet from '../../lib/utilities/deepGet';
+import formatDuration from '../../lib/utilities/formatDuration';
+import { storyItem } from '../../models/propTypes/storyItem';
+
+const LinkContents = ({ item }) => {
+  const {
+    translations: { media: mediaTranslations },
```

Comment:
Will this blow up if `mediaTranslations` is missing? Ive not done much nested destructuring before :P

Reply:
Woah it doesnt, coool